awx/api/serializers.py

```diff
@@ -2175,14 +2175,25 @@ def get_related(self, obj):
 
     def get_summary_fields(self, obj):
         summary_fields = super(InventoryUpdateDetailSerializer, self).get_summary_fields(obj)
-        summary_obj = self.get_source_project(obj)
 
-        if summary_obj:
+        source_project = self.get_source_project(obj)
+        if source_project:
             summary_fields['source_project'] = {}
             for field in SUMMARIZABLE_FK_FIELDS['project']:
-                value = getattr(summary_obj, field, None)
+                value = getattr(source_project, field, None)
                 if value is not None:
                     summary_fields['source_project'][field] = value
+
+        cred = obj.credentials.first()
+        if cred:
+            summary_fields['credential'] = {
+                'id': cred.pk,
+                'name': cred.name,
+                'description': cred.description,
+                'kind': cred.kind,
```

Comment:
we will most likely be removing `kind` in the near future but we haven't actually done so _yet_ - because of this i've kept the `kind` field in the summary to keep it consistent with how we represent summarized credentials elsewhere.